### PR TITLE
fix(ui5-color-picker): adjust hue value update when user presses over the main color section

### PR DESCRIPTION
**Pull request by unazko**
Fixes:  #4540

### CODE REVIEW

**Comment by tsanislavgatev** [comment]
I think this refered issue is not fixed yet: https://github.com/SAP/ui5-webcomponents/issues/4541
Example:
I input ffffff, then pick red from slider and move the picker to a place different than bottom right corner(white), and then input ffffff again, the color change is not applied.

**Review by tsanislavgatev — CHANGES_REQUESTED** [review]
I think this refered issue is not fixed yet: #4541
Example:
I input ffffff, then pick red from slider and move the picker to a place different than bottom right corner(white), and then input ffffff again, the color change is not applied.

**Comment by unazko** [comment]
You are right @tsanislavgatev,
I've created a separate PR for the other issue:
https://github.com/SAP/ui5-webcomponents/pull/4621
as it feels more suitable now.

**Review by tsanislavgatev — DISMISSED** [review]

File: packages/main/test/specs/ColorPicker.spec.js [hunk 1]
```diff
@@ -102,4 +102,20 @@ describe("Color Picker general interaction", () => {
...
+
+		await hexInput.doubleClick();
+		await browser.keys("0a6ed1");
+		await browser.keys("Enter");
+
+		const hueValue = await colorPicker.getAttribute("_hue");
+
+		await mainColorSection.click();
+
+		assert.strictEqual(await colorPicker.getAttribute("_hue"), hueValue, "Hue value remained unchanched");
```
Comment (ilhan007) [inline packages/main/test/specs/ColorPicker.spec.js:118]: typo: "unchanched" -> "unchanged"

**Review by ilhan007 — DISMISSED** [review]
Again, as in the other PR I reviewed - please pay attention on the commit message title https://www.conventionalcommits.org/en/v1.0.0/#examples

File: packages/main/test/specs/ColorPicker.spec.js (hunk 1 above)
Comment (unazko) [reply packages/main/test/specs/ColorPicker.spec.js]: Done

**Review by tsanislavgatev — APPROVED** [review]

**Pull request merged**